io.github.ec_.Quake3e.OpenArena.yaml

```diff
@@ -14,7 +14,7 @@ finish-args:
   # Needs a persistent location for Quake 3 data files, the user
   # has permission to add things to and this is the only one in Quake3e's
   # search path.
-  - --persist=.q3a
+  - --persist=.var/app/io.github.ec_.Quake3e.OpenArena/data
```

Comment:
This should not be needed (and should not be set). The XDG data dir should be persistant by default, as far as I know.